fuzz/init.sh

```diff
@@ -9,7 +9,7 @@ lib=gfoidl.Base64.dll
 
 dotnet build -c Release gfoidl.Base64.FuzzTests
 mkdir -p ./instrumented
-cp ../source/gfoidl.Base64/bin/Release/netcoreapp3.0/$lib ./instrumented/$lib
+cp ../source/gfoidl.Base64/bin/Release/netcoreapp3.1/$lib ./instrumented/$lib
```

Comment:
```suggestion
cp ../source/gfoidl.Base64/bin/Release/netcoreapp3.0/$lib ./instrumented/$lib
```
This should remain as the lib is built with .NET Core 3.0 target (for broader usage).